frontend/src/styles/css-variables.ts

```
@@ -0,0 +1,341 @@
+  
+  // Colors
+  cssVars.push('/* Colors */');
+  cssVars.push(...objectToCSSVariables(designSystem.colors, 'color'));
+  cssVars.push('');
+  
+  // Typography
+  cssVars.push('/* Typography */');
+  cssVars.push(...objectToCSSVariables({
+    fontFamily: designSystem.typography.fontFamily,
+    fontSize: Object.entries(designSystem.typography.fontSize).reduce((acc, [key, value]) => ({
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)